rfcs/20180907-contrib-sunset.md

```diff
@@ -0,0 +1,282 @@
+to a repo tentatively maintained by a SIG, but for which no SIG is actually
+formed) will be deleted by default at 2.0.
+
+| Project                 | Owner(s)    | Fate                                 |
+|:----------------------- |:----------- |:------------------------------------ |
+| tools/graph_transforms  | petewarden  | delete (replaced by grappler)        |
+| tools/quantization | petewarden       | delete (part of tflite)              |
+| all_reduce         | poxvoculi   | delete (replaced by DistributionStrategy) |
+| android            | petewarden       | delete (redundant: tflite/ARM build) |
+| autograph          | alexbw           | move to core                         |
+| batching           | chrisolston      | delete                               |
+| bayesflow          | jvdillon     | delete (moved to tensorflow/probability) |
+| benchmark_tools    | saeta            | delete                               |
+| bigtable           | saeta            | move to tf.data                      |
+| boosted_trees     | nataliaponomareva | delete (moved to core)               |
+| checkpoint         | allenlavoie      | move to core                         |
+| cloud              | saeta            | merge with bigtable/                 |
+| cluster_resolver   | frankchn         | move to core                         |
+| cmake              | mrry             | delete (replaced with bazel)         |
+| coder   | nickj-google jonycgn sungjinhwang | move to tensorflow/compression |
+| compiler           | tatatodd         | move to core                         |
+| constrained_optimization | acotter    | move to separate repo                |
+| copy_graph         |                  | delete (no owner)                    |
+| crf                |                  | merge into tensorflow/probability?   |
+| cudnn_rnn          |                  | delete (available in tf.keras.layers)|
+| data               | mrry             | delete (moved to core)               |
+| decision_trees     | salehay          | delete                               |
+| deprecated         |                  | delete (deprecated)                  |
+| distribute         | josh11b priyag   | move to core                         |
+| distributions      | jvdillon     | delete (moved to tensorflow/probability) |
+| eager              | asimshankar      | move to core                         |
+| estimator          | ispirmustafa mikecase | move to tensorflow/estimator    |
+| factorization      | agarwal-ashish   | delete (rebuild in core pending)?    |
+| feature_column     | ispirmustafa     | move to core (experimental)          |
+| ffmpeg             | fredbertsch      | delete                               |
+| framework          |               | partially move to core, delete the rest |
+| fused_conv         |                  | delete                               |
+| gan                | joel-shor        | move to separate repo                |
```

Comment:
@galeone TFGAN will be moved to a separate repo. You can find more details here.